Cargo.toml

```diff
@@ -78,6 +83,7 @@ backend_egl = ["gl_generator", "libloading"]
 backend_libinput = ["input"]
 backend_session = []
 backend_udev = ["udev", "input/udev"]
+backend_vulkan = ["backend_drm", "ash", "scopeguard"]
```

Comment:
I assume a `renderer_vulkan` would not need `backend_drm`, no? Does `backend_vulkan` strictly need it?

Reply:
backend_drm is a result of DrmNode being used. Unless we wanted to split that out into another crate or feature there is no real way around it.